packages/castmill/lib/castmill/addons/playlists/components/playlist-item.tsx

```diff
@@ -40,13 +40,42 @@ const Thumbnail: Component<{
 }> = (props) => {
   const thumbnailUri = getThumbnailUri(props.item);
   const widgetName = getWidgetName(props.item);
+  const widgetIcon = props.item.widget.icon;
+
+  const isImageIcon =
+    widgetIcon &&
+    (widgetIcon.startsWith('data:image/') ||
+      widgetIcon.startsWith('http://') ||
+      widgetIcon.startsWith('https://') ||
+      widgetIcon.startsWith('/'));
 
   return (
     <Show
       when={thumbnailUri}
-      fallback={<div>{widgetName}</div>}
+      fallback={
+        <div class={styles.widgetIcon}>
+          <Show
+            when={isImageIcon}
+            fallback={<span>{widgetIcon || '📦'}</span>}
```

Comment:
Since '📦' is used twice here (and possibly elsewhere as well it could make sense to store it in a const instead of inlining it.